emails/views.py

```diff
@@ -329,6 +333,7 @@ def _get_text_and_html_content(email_message):
     else:
         if email_message.get_content_type() == 'text/plain':
             text_content = email_message.get_content()
+            html_content = email_message.get_content()
```

Comment:
Dang. Setting the content to `text/html` breaks the `\n` line breaks in the plain text part. 😢 (from a reddit signup email)

<img width="1568" alt="image" src="https://user-images.githubusercontent.com/71928/85091895-24ea9280-b1ae-11ea-875d-2289198ecd1f.png">

I'm not sure if we should try to go thru the text content and replace `\n` with `<br>` or not? There might be other things that break when plain text is rendered as html.